.github/workflows/ansible-lint.yml

```
@@ -15,6 +15,14 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      # Python version being set because of issue with Ubuntu permissions and versions of Python < 3.11 
+      - name: Set up Python
+        # This is the version of the action for setting up Python, not the Python version.
+        uses: actions/setup-python@v5
+        with:
+          # Semantic version range syntax or exact version of a Python version
+          python-version: '3.11' 
+
```

We should:
a) link to the issue
b) tighten up the comments a bit
c) pin to py 3.12 (since that's what we were on before)
```suggestion
      # Python version must be pinned because of issue with Ubuntu permissions
      # See https://github.com/actions/runner-images/issues/11499  
      - name: Set up Python
        uses: actions/setup-python@v5
        with:
          python-version: '3.12' 

```